Frontend/ui-customer/src/Services/API.purs

```diff
@@ -590,7 +590,9 @@ newtype EstimateAPIEntity = EstimateAPIEntity {
   totalFareRange :: Maybe FareRange,
   nightShiftRate :: Maybe NightShiftRate,
   specialLocationTag :: Maybe String,
-  driversLatLong :: Array LatLong
+  driversLatLong :: Array LatLong,
+  serviceTierName :: Maybe String,
+  serviceTierShortDesc :: Maybe String
```

Comment:
please replace `serviceTierName ` with `variantCategory` and `serviceTierShortDesc ` with `variantCategoryDescription` at all the places